engine/api/user.go

```diff
@@ -155,6 +158,48 @@ func (api *API) getUsersHandler() Handler {
 	}
 }
 
+// postUserFavoriteHandler post favorite user for workflow or project
+func (api *API) postUserFavoriteHandler() Handler {
```

Comment:
please add unit test

Reply:
maybe he does not have this linter : https://github.com/fsamin/nofuncflags